keyboards/silverbullet44/keymaps/default/keymap.c

```diff
@@ -0,0 +1,214 @@
+/* Copyright 2019 SwanMatch
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines the keycodes used by our macros in process_record_user
+//enum custom_keycodes {
+//  QMKBEST = SAFE_RANGE,
+//  QMKURL
+//};
+enum layer {
+    _QWERTY,
+    _CURSOL,
+    _CALC,
+    _ADJUST,
+};
+
+enum custom_keycodes {
+  ADJUST = SAFE_RANGE,
+  RGBRST,
+  KC_00,
+  ALTAB
+};
+#define CALC LT(_CALC,   KC_ESC)
+#define CUSL LT(_CURSOL, KC_TAB)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* QWERTY
+ * ,-----------------------------------------.                                  ,-----------------------------------------.
+ * |TabMOS|   Q  |   W  |   E  |   R  |   T  |                                  |   Y  |   U  |   I  |   O  |   P  |  =   |
+ * |------+------+------+------+------+------|                                  |------+------+------+------+------+------|
+ * |Shift |   A  |   S  |   D  |   F  |   G  |                                  |   H  |   J  |   K  |   L  |   ;  |  '   |
+ * |------+------+------+------+------+------|                                  |------+------+------+------+------+------|
+ * | Ctrl |   Z  |   X  |   C  |   V  |   B  |-------.-------.  ,---------------|   N  |   M  |   ,  |   .  |   /  |  -   |
+ * `-----------------------------------------/       /       /   \       \      \----------------------------------------'
+ *                          | Alt  | Bksp | / Shift / Calc  /     \ Cursol\ Ctrl | Del  |  App  |
+ *                          |  F5  |      |/ Space /  Esc  /       \  Tab  \ Enter\     |  F12  |
+ *                          `-----------------------------'         '---------------------------'
+ */
+  [_QWERTY] = LAYOUT( \
+    KC_TAB,    KC_Q, KC_W, KC_E, KC_R, KC_T,                KC_Y,   KC_U,    KC_I,    KC_O,   KC_P,    KC_EQL,  \
+    KC_LSFT,   KC_A, KC_S, KC_D, KC_F, KC_G,                KC_H,   KC_J,    KC_K,    KC_L,   KC_SCLN, KC_QUOT, \
+    KC_LCTRL,  KC_Z, KC_X, KC_C, KC_V, KC_B,                KC_N,   KC_M,    KC_COMM, KC_DOT, KC_SLSH, KC_MINS, \
+     LALT_T(KC_F5), KC_BSPC, LSFT_T(KC_SPC), CALC,    CUSL, LCTL_T(KC_ENT), KC_DEL, GUI_T(KC_F12) \
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.